axelrod/strategies/worse_and_worse.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+from axelrod import Actions, Player, random_choice
+
+C, D = Actions.C, Actions.D
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / 1000'. Therefore
+    it is more and more likely to defect as the round goes on.
+
+    Names:
+        - worse_and_worse: [PRISON1998]
+
+    """
+
+    name = 'Worse and worse'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+        if random.randint(0, 1000) < current_round:
+            return D
+        return C
+
+
+class WorseAndWorseRandom (Player):
```

Comment:
Not a fan of this name: it makes it sound like `WorseAndWorse` is not random. As this is a version of `WorseAndWorse` that has some knowledge I suggest you call it `KnowledgeableWorseAndWorse`.